package.json

```diff
@@ -121,7 +122,6 @@
       "toggle-off",
       "traffic-cone",
       "trash-can",
-      "up-right-and-down-left-from-center",
```

Comment:
Verified the `far` version of these are no longer used anywhere else in the app.